### PR TITLE
Add FromScratch v1.0.1

### PR DESCRIPTION
**Pull request by Luzifer**
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/fromscratch.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'fromscratch' do
+  version '1.0.1'
+  sha256 '6c3d25a26bca0ab018f808856afb09adfbb14c622e7aff69e47f6d26e6912036'
+
+  url 'https://github.com/Kilian/fromscratch/releases/download/v1.0.0/FromScratch-darwin-x64-1.0.1.zip'
```
Comment (vitorgalvao) [inline Casks/fromscratch.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

Comment (vitorgalvao) [inline Casks/fromscratch.rb]: url "https://github.com/Kilian/fromscratch/releases/download/v#{version}/FromScratch-darwin-x64-#{version}.zip"`

Comment (vitorgalvao) [inline Casks/fromscratch.rb]: Needs a `# github.com/Kilian/fromscratch was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

**Comment by Luzifer** [comment]
Added appcast and verification comment. Sadly the `#{version}` variable is not usable in that URL as the versions differ for the OSX artifact (`v1.0.0` vs. `1.0.1`).

@vitorgalvao PTAL again

**Pull request merged**